apps/forklift/lib/forklift/migrations.ex

```diff
@@ -1,92 +1,46 @@
 defmodule Forklift.Migrations do
   @moduledoc """
-  Handle forklift view state migrations
+  Contains all migrations that run during startup.
   """
-  use GenServer, restart: :transient
-
   require Logger
-  import SmartCity.Event, only: [data_ingest_start: 0]
-
-  def start_link(opts) do
-    GenServer.start_link(__MODULE__, opts)
-  end
-
-  def init(_opts) do
-    Logger.info("#{__MODULE__}: Beginning forklift migration")
-    {:ok, pid} = start_brook()
-
-    migrate_datasets_to_process()
+  use GenServer, restart: :transient
 
-    stop_brook(pid)
-    Logger.info("#{__MODULE__}: forklift migration completed")
-    {:ok, :ok, {:continue, :stop}}
-  end
+  import Forklift, only: [instance_name: 0]
 
-  defp migrate_datasets_to_process() do
-    Brook.get_all!(:migration, :datasets_to_process)
-    |> Enum.map(fn {key, value} -> %{key: key, old: value} end)
-    |> Enum.map(fn %{key: key} = entry -> Map.put(entry, :dataset, get_dataset(key)) end)
-    |> Enum.map(fn %{key: key} = entry -> Map.put(entry, :events, get_events(key)) end)
-    |> Enum.each(&add_entry/1)
+  @instance Forklift.instance_name()
 
-    rename_old_entries()
+  def start_link(args) do
+    GenServer.start_link(__MODULE__, args, name: __MODULE__)
   end
 
-  defp add_entry(entry) do
-    Enum.each(entry.events, fn event ->
-      Brook.Test.with_event(:migration, event, fn ->
-        Brook.ViewState.merge(:datasets, entry.dataset.id, entry.dataset)
-      end)
-    end)
-  end
+  def init(_args) do
+    migrate_once("last_insert_date_completed", "migration:last_insert_date:start")
 
-  defp get_dataset(id) do
-    get_events(id)
-    |> Enum.filter(fn event -> event.type == data_ingest_start() end)
-    |> List.last()
-    |> Map.get(:data)
+    {:ok, :ok, {:continue, :stop}}
   end
 
-  defp get_events(id) do
-    Brook.get_events!(:migration, :datasets_to_process, id)
+  def handle_continue(:stop, state) do
+    {:stop, :normal, state}
   end
 
-  defp rename_old_entries() do
-    client = Forklift.Application.redis_client()
-
-    case Redix.command!(client, ["KEYS", "forklift:view:*:datasets_to_process:*"]) do
-      [] ->
-        :ok
-
-      keys ->
-        commands =
-          Enum.reduce(keys, [], fn key, acc ->
-            acc ++
-              [
-                ["RENAME", key, "old:" <> key],
-                ["EXPIRE", "old:" <> key, "600000"]
-              ]
-          end)
-
-        Redix.pipeline!(client, commands)
+  def migrate_once(completed_flag_name, event_name) do
+    Patiently.wait_for(
```

Comment:
We need to put in a PR on Elsa and/or Brook to get this fixed ASAP

Reply:
Actual issue was having config: where it should have been consumer_config in the integration test. Fixed.